### DISCUSSION TITLE
does `.md` files has possible to show preview mode?

### DISCUSSION THREAD

**Discussion by aliuq** [community/community#12874] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi

For markdown file, it always means documetation here, I want to custom my scope search rules, e.g. groups with vuejs and vitejs docs

Does **.md** files has possible to show preview mode?

Thank you.

**Comment by colinwm** [comment, marked as answer]
> I want to custom my scope search rules, e.g. groups with vuejs and vitejs docs

Do you mean you want to find markdown files in your custom scope? Or exclude them? Either way, you can do something like:

`... AND NOT language:markdown` 

or 

`... path:*.md`  

to include/exclude markdown files.

> Does .md files has possible to show preview mode?

Right now, we don't show any previews within cs.github.com, but you can always click on the "view on github" link to see a preview on github.com. And FYI, the keyboard shortcut for "view on github.com" is `,`.

**Reply by aliuq** [reply]
Thank you for reply, I mean the second, because of many repo's documentaion resource all in here, so I want to search all in one page.
